src/api/users.coffee

```diff
@@ -60,32 +136,31 @@ exports.getNewUser = (token) ->
     else
       # if expiry date has past
       if moment(result.expiry).utc().format() < moment().utc().format()
```

Comment:
I know this is existing code, but it might better to just do something like `if moment(result.expiry).isBefore(moment())`